_hardcoded/tracing.go

```diff
@@ -84,10 +84,10 @@ func newTracerProvider(exporter sdktrace.SpanExporter, resource *resource.Resour
 			EventCountLimit:     100,
 			LinkCountLimit:      100,
 		}),
-		//Batcher is more efficient, switch to it after testing
+
+		// Batcher is more efficient, switch to it after testing
 		sdktrace.WithSyncer(exporter),
-		//sdktrace.WithBatcher(exporter),
-		//Have to figure out how I'm going to generate this resource first.
+		// sdktrace.WithBatcher(exporter),
```

Comment:
Does this need to get changed back?

Reply:
I think it might be worth doing at this point. FYI, we've had it set to WithSyncer in prod this whole time. My only worry is that while we are testing on low use services it might take a while for it to form a big enough batch to send to the collector, making testing harder.

Reply:
Ah I see, if you're only on low use services for now and this is intended then I'm not worried about leaving this in. The extra context helps. Thanks!